apps/server/src/modules/tldraw/service/tldraw.ws.service.spec.ts

```diff
@@ -224,38 +265,51 @@ describe('TldrawWSService', () => {
 			};
 
 			it('should call send method when received message of type SYNC', async () => {
-				const { sendSpy, applyAwarenessUpdateSpy, syncProtocolUpdateSpy, doc, msg } = await setup([0, 1]);
+				const { sendSpy, applyAwarenessUpdateSpy, syncProtocolUpdateSpy, doc, msg } = await setup([0, 1], false);
```

Comment:
Please do not activate of deactivate mocks with passing params. The setup is unique for the when description. Throwing a error or not are completly different senarios with different when descriptions and different setups. The goal is that the when descriptions really describe what it is happen in setup.